docs/auth0_api.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+---
+## auth0 api
+
+Makes an authenticated HTTP request to the Auth0 Management API
+
+### Synopsis
+
+Makes an authenticated HTTP request to the Auth0 Management API and prints the response as JSON.
+
+The method argument is optional, and when you don’t specify it, the command defaults to GET for requests without data and POST for requests with data.
+
+Auth0 Management API Docs:
+  https://auth0.com/docs/api/management/v2
+
+Available Methods:
+  GET, POST, PUT, PATCH, DELETE
+
+```
+auth0 api <method> <uri> [flags]
+```
+
+### Examples
+
+```
+auth0 api "/stats/daily?from=20221101&to=20221118"
+auth0 api get "/tenants/settings"
```

Comment:
Is the forward slash in the path (e.g. "/tenants/settings") necessary? Seems like all the path args will need it, so maybe we can dispense from having to include it?

Reply:
It isn't necessary, it can be there or not, depending on the user's preference: https://github.com/auth0/auth0-cli/pull/531/files#diff-9641610d0216306cf67e4b06eecff4cc3ec029d1bdbe4b53a6371b20ccdd9846R170 as we trim extra trailing slashes.

Reply:
No, I'm suggesting to not include it in the examples if it's not necessary.

Reply:
Updated in: [`304faa1` (#531)](https://github.com/auth0/auth0-cli/pull/531/commits/304faa15f1e5d4c6b2ba2a4729f878ed4b1cccf7)